### DISCUSSION TITLE
Tiled VAE is not needed on Mac

### DISCUSSION THREAD

**Discussion by JeoChen** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#146] [Q&A] [answered]
it doesnt work in mac

**Comment by pkuliyi2015** [comment]
After consideration I found it necessary to enable on Mac. But I just tested positive for code nineteen, so i don't have time to do that. I will Make updates. After recovery

**Reply by Kahsolt** [reply, marked as answer]
This feature should be available now. check commit 5da750b9de930b0e28883423f697c5ea82457c24
